bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/RuntimeUtils.java

```diff
@@ -150,13 +150,18 @@ private static void printToConsole(ErrorValue throwable) {
     }
 
     public static void handleRuntimeReturnValues(Object returnValue) {
-        if (returnValue instanceof ErrorValue) {
-            ErrorValue errorValue = (ErrorValue) returnValue;
+        if (returnValue instanceof ErrorValue errorValue) {
             errStream.println("error: " + errorValue.getPrintableError());
             Runtime.getRuntime().exit(1);
         }
     }
 
+    public static void handleRuntimeErrorReturns(Object returnValue) {
+        if (returnValue instanceof ErrorValue errorValue) {
```

Comment:
Can we have any other value than ErrorValue. What happen if we have any other value?

Reply:
Removed this unused code block since when there is a error return we need to terminate the program execution by returning the error value.

Reply:
Restored this method since it's needed for give the console output when errors are returned from the `gracefulStop()` methods of dynamic listeners and the functions registered with the `runtime:onGracefulStop`. 



> Can we have any other value than ErrorValue. What happen if we have any other value?

Yes, it's actually the return values of the functions. If there a error return from the main, the error message will be added to the console output and exit the program gracefully. If an error returns from any function which is a part of the shutdown sequence (where the user do not have the control to handle the error returns from their end), the error message also needed to be logged into the console similar to the error returns of main.